### PR TITLE
Ensure you are on latest master before running update translations

### PR DESCRIPTION
**Pull request by proteusvacuum**
I wasn't on latest master when running this and ran into issues. 

@dannyroberts 
@sravfeyn

### CODE REVIEW

**Review by kaapstorm — APPROVED** [review]
Nice. This has caught me before too.

File: scripts/update-translations.sh [hunk 1]
```diff
@@ -22,6 +22,14 @@ then
     abort "You must be on master to run this command."
 fi
 
+git fetch
+tracking_branch=`git rev-parse --abbrev-ref --symbolic-full-name @{u}`
+upstream_changes=`git rev-list HEAD...$tracking_branch --count`
```
Comment (dannyroberts) [inline scripts/update-translations.sh]: interesting, so this checks whether there are upstream changes on whatever branch you're on, but not that you're on branch master. Any particular reason for that?

**Review by dannyroberts — APPROVED** [review]
Nice, very pro pushing any kind of best-practice process into our tools like this!

File: scripts/update-translations.sh (hunk 1 above)
Comment (proteusvacuum) [reply scripts/update-translations.sh]: The previous check aborts if you are not on master.

Comment (proteusvacuum) [reply scripts/update-translations.sh]: I did it this way in case anyone has the upstream remote not set called `origin`

**Pull request merged**